src/config/theme.ts

```diff
@@ -0,0 +1,225 @@
+import type { ThemeConfig } from "./type";
```

Comment:
create theme folder containing
- index.ts (importing DEFAULT_THEME from default.ts and OSMOSIS_THEME from osmosis.ts)
- default.ts
- osmosis.ts